tasks/remove-tags.js

```
@@ -0,0 +1,254 @@
+                            try {
+                                // Filter out the tags that need to be removed
+                                const tagsToRemove = ctx.args.remove_tags;
+                                
+                                let updatedTags;
+                                if (!tagsToRemove || tagsToRemove.length === 0) {
+                                    // If no tags specified (leave blank for all), remove all tags
+                                    updatedTags = [];
+                                } else {
+                                    // Handle both string array (CLI) and object array (interactive) formats
+                                    const tagNamesToRemove = tagsToRemove.map(tag => 
+                                        typeof tag === 'string' ? tag : tag.name
```

ESLint

[error] Arrow function used ambiguously with a conditional expression.

(no-confusing-arrow)